zerver/models/streams.py

```diff
@@ -127,6 +127,10 @@ class Stream(models.Model):
     # when realm itself is deleted.
     can_remove_subscribers_group = models.ForeignKey(UserGroup, on_delete=models.RESTRICT)
 
+    can_administer_channel_group = models.ForeignKey(
+        UserGroup, on_delete=models.RESTRICT, related_name="+"
+    )
```

Comment:
Probably the `on_delete` comment could use some rewording to replace "here" with "for group-valued settings". Also probably wise to put this above the existing one.